string label
string symbol_path
uint8 score
int16 x1
int16 y1
int16 x2
int16 y2
